cfme/configure/configuration/region_settings.py

```diff
@@ -1038,6 +1037,9 @@ def set_replication(self, updates=None, replication_type=None, reset=False):
             view.flash.assert_message('All changes have been reset')
         else:
             try:
+                if validate:
+                    view.action_dropdown.item_select("Validate")
+                    view.flash.assert_message("Subscription Credentials validated successfully")
                 view.accept_button.click()
```

Comment:
can we use ```assert_success_message``` here?

Reply:
yes, now used  assert_success_message() .